{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "low"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for graphviz",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for graphviz fixes the following issues:\n\n- CVE-2018-10196: Fixed a null dereference in rebuild_vlis (bsc#1093447).\t  \n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "slehasp4-graphviz-14524",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2020_14524-1.json"
			},
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2020:14524-1",
				"url": "https://www.suse.com/support/update/announcement/2020/suse-su-202014524-1/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2020:14524-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2020-October/007677.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1093447",
				"url": "https://bugzilla.suse.com/1093447"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2018-10196 page",
				"url": "https://www.suse.com/security/cve/CVE-2018-10196/"
			}
		],
		"title": "Security update for graphviz",
		"tracking": {
			"current_release_date": "2020-10-30T09:47:25Z",
			"generator": {
				"date": "2020-10-30T09:47:25Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2020:14524-1",
			"initial_release_date": "2020-10-30T09:47:25Z",
			"revision_history": [
				{
					"date": "2020-10-30T09:47:25Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "graphviz-python-2.20.2-8.3.6.i586",
								"product": {
									"name": "graphviz-python-2.20.2-8.3.6.i586",
									"product_id": "graphviz-python-2.20.2-8.3.6.i586"
								}
							}
						],
						"category": "architecture",
						"name": "i586"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "graphviz-python-2.20.2-8.3.6.ppc64",
								"product": {
									"name": "graphviz-python-2.20.2-8.3.6.ppc64",
									"product_id": "graphviz-python-2.20.2-8.3.6.ppc64"
								}
							}
						],
						"category": "architecture",
						"name": "ppc64"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "graphviz-python-2.20.2-8.3.6.s390x",
								"product": {
									"name": "graphviz-python-2.20.2-8.3.6.s390x",
									"product_id": "graphviz-python-2.20.2-8.3.6.s390x"
								}
							}
						],
						"category": "architecture",
						"name": "s390x"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "graphviz-python-2.20.2-8.3.6.x86_64",
								"product": {
									"name": "graphviz-python-2.20.2-8.3.6.x86_64",
									"product_id": "graphviz-python-2.20.2-8.3.6.x86_64"
								}
							}
						],
						"category": "architecture",
						"name": "x86_64"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Linux Enterprise High Availability Extension 11 SP4",
								"product": {
									"name": "SUSE Linux Enterprise High Availability Extension 11 SP4",
									"product_id": "SUSE Linux Enterprise High Availability Extension 11 SP4",
									"product_identification_helper": {
										"cpe": "cpe:/a:suse:sle-hae:11:sp4"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "graphviz-python-2.20.2-8.3.6.i586 as component of SUSE Linux Enterprise High Availability Extension 11 SP4",
					"product_id": "SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.i586"
				},
				"product_reference": "graphviz-python-2.20.2-8.3.6.i586",
				"relates_to_product_reference": "SUSE Linux Enterprise High Availability Extension 11 SP4"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "graphviz-python-2.20.2-8.3.6.ppc64 as component of SUSE Linux Enterprise High Availability Extension 11 SP4",
					"product_id": "SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.ppc64"
				},
				"product_reference": "graphviz-python-2.20.2-8.3.6.ppc64",
				"relates_to_product_reference": "SUSE Linux Enterprise High Availability Extension 11 SP4"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "graphviz-python-2.20.2-8.3.6.s390x as component of SUSE Linux Enterprise High Availability Extension 11 SP4",
					"product_id": "SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.s390x"
				},
				"product_reference": "graphviz-python-2.20.2-8.3.6.s390x",
				"relates_to_product_reference": "SUSE Linux Enterprise High Availability Extension 11 SP4"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "graphviz-python-2.20.2-8.3.6.x86_64 as component of SUSE Linux Enterprise High Availability Extension 11 SP4",
					"product_id": "SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.x86_64"
				},
				"product_reference": "graphviz-python-2.20.2-8.3.6.x86_64",
				"relates_to_product_reference": "SUSE Linux Enterprise High Availability Extension 11 SP4"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2018-10196",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2018-10196"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "NULL pointer dereference vulnerability in the rebuild_vlists function in lib/dotgen/conc.c in the dotgen library in Graphviz 2.40.1 allows remote attackers to cause a denial of service (application crash) via a crafted file.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.i586",
					"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.ppc64",
					"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.s390x",
					"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2018-10196",
					"url": "https://www.suse.com/security/cve/CVE-2018-10196"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1093447 for CVE-2018-10196",
					"url": "https://bugzilla.suse.com/1093447"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.i586",
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.ppc64",
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.s390x",
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 3.3,
						"baseSeverity": "LOW",
						"vectorString": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:L",
						"version": "3.0"
					},
					"products": [
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.i586",
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.ppc64",
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.s390x",
						"SUSE Linux Enterprise High Availability Extension 11 SP4:graphviz-python-2.20.2-8.3.6.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2020-10-30T09:47:25Z",
					"details": "low"
				}
			],
			"title": "CVE-2018-10196"
		}
	]
}
